### ISSUE TITLE
Consultatation: Group Photo

### ISSUE THREAD

**Issue by NGRhodes** [ARCLeeds/arcleeds.github.io#26]
I'd put a group photo somewhere on this page, if you're willing. It would make you more approachable.

**Comment by NGRhodes** [comment]
Good idea, but a challenge to do now, maybe collate teams photos - might be worth putting on about page.

**Comment by Sparrow0hawk** [comment]
Think we can definitely park this one for the foreseeable future.
